src/PRASBase/pras_system_size_reducer.jl

```diff
@@ -0,0 +1,91 @@
+#######################################################
+# Surya
+# NREL
+# January 2020
+# Make a smaller PRAS system (with a fewer timesteps)
+# from a bigger PRAS system
+#######################################################
+# Main Function to reduce size of PRAS System
+#######################################################
+function reduce_pras_system_size(pras_system::PRAS.SystemModel,period_of_interest::UnitRange)
```

Comment:
Could this just be a constructor method? i.e

```julia
SystemModel(system::SystemModel, period_of_interest::UnitRange)
```

Also, would it be more useful for the starting and ending points to be provided as timestamps instead of integers? So

```julia
SystemModel(system::SystemModel, start::ZonedDateTime, stop::ZonedDateTime)
```

Reply:
I implemented both the methods in my latest commit